downloads/aws/setup-aws.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+check_aws_connection(){
+	print_function_details
+	CALLER_ARN=$(aws sts get-caller-identity | jq -r '.Arn')
+	if [[ -z "$CALLER_ARN" ]]; then	
+		print_error_and_exit "The aws cli does not have the propper credentials. Please fix and then try again."
+	fi
+}
+
+parse_user_input(){
+	print_function_details
+	AWS_ACCOUNT_NAME=$(jq -r '.AWS_ACCOUNT_NAME' fill-me-out.json)
```

Comment:
Is jq installed on most systems?

Reply:
No? But I'm using it for now. Again, I want to re-write this in terraform in the future to make things easier. But, for now, I need something to parse the json returned from Amazon, and I've decided on `jq`.

Updated the readme to call out this as a requirement, which I forgot to do before. This will be translated into a pretty page as described above.